openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -20,18 +20,32 @@ def create_taxonomy(
     enabled=True,
     allow_multiple=False,
     allow_free_text=False,
+    orgs: list[Organization] = [],
 ) -> Taxonomy:
     """
     Creates, saves, and returns a new Taxonomy with the given attributes.
     """
-    return oel_tagging.create_taxonomy(
+    taxonomy = oel_tagging.create_taxonomy(
         name=name,
         description=description,
         enabled=enabled,
         allow_multiple=allow_multiple,
         allow_free_text=allow_free_text,
     )
 
+    if orgs:
+        TaxonomyOrg.objects.bulk_create(
+            [
+                TaxonomyOrg(
+                    taxonomy=taxonomy,
+                    org=org,
+                    rel_type=TaxonomyOrg.RelType.OWNER,
+                )
+                for org in orgs
+            ]
+        )
```

Comment:
We've already got an api call for this:

```suggestion
        set_taxonomy_orgs(taxonomy, all_orgs=False, orgs=orgs)
```

Reply:
Done 410dfe76033b30ff401fa2ab065c3514bb2a7ae7